cypress/support/utils/fileManager.js

```diff
@@ -3,9 +3,15 @@ const downloadsFolder = Cypress.config('downloadsFolder');
 export default {
   findDownloadedFilesByMask(mask) {
     // more about masks: https://github.com/isaacs/minimatch#usage
+    // returns array with all files matched to mask
     return cy.task('findFiles', `${downloadsFolder}/${mask}`);
   },
 
+  getLastDownloadedFileName(downloadedFilenames) {
+    // returns only one last downloaded file from array of files
+    return downloadedFilenames.sort()[downloadedFilenames.length - 1];
```

Comment:
try to work with lastIndexOf or index -1 of array. Additionally, in case of opposite order you can take first element